src/app/helpers/elements.ts

```diff
@@ -14,7 +14,7 @@ export interface TypeParams {
 export const getCurrentType = ({ labelID, mailSettings }: TypeParams) =>
     isConversationMode(labelID, mailSettings) ? ELEMENT_TYPES.CONVERSATION : ELEMENT_TYPES.MESSAGE;
 
-export const isConversation = (element: Element): boolean => !(element as Message).conversationID;
+export const isConversation = (element: Element): boolean => !(element as Message).ConversationID;
 export const isMessage = (element: Element): boolean => !isConversation(element);
```

Comment:
I feel it's better to test if the `element` is a message by checking the presence of  `ConversationID` instead of using a negative condition.

``` js
export const isMessage = (element: Element): boolean => (element as Message).ConversationID;
```

Reply:
Yep, less negation is always better. I missed that one because isConversation is so much more used that isMessage.